File: src/main/resources/js/HelloController.js
import request from 'axios';export function hello() { return request({   url: '/hello/helloGet',   method: 'get',    });}export function helloNotRestful(id, name) { return request({   url: '/hello/helloNotRestful',   method: 'get',   param: {     id: id,     name: name,   } });}export function helloPost(body) { return request({   url: '/hello/helloPost',   method: 'post',   data: {     body: body,   } });}export function helloGetById(id) { return request({   url: '/hello/helloGet/'+id,   method: 'get',    });}

File: assistant-plus-generator/src/main/java/io/github/controller/HelloController.java
package io.github.controller;

import org.springframework.web.bind.annotation.*;

/**
 * @author Genius
 *
 **/
@RestController
@RequestMapping("/hello")
public class HelloController {

    @GetMapping("/helloGet")
    public void hello(){
        System.out.println("hello");
    }

    @PostMapping("/helloPost")
    public void helloPost(@RequestBody String body){
        System.out.println("helloPost");
    }

    @GetMapping("/helloGet/{id}/{name}")
    public void helloGetById(@PathVariable("id") String id,@PathVariable("name") String name){
        System.out.println("helloGet");
    }

    @GetMapping("/helloNotRestful")
    public void helloNotRestful(@RequestParam("id") String id,@RequestParam("name") String name){
        System.out.println("helloNotRestful");
    }

    @RequestMapping(value = "/helloRequest",method = {RequestMethod.GET,RequestMethod.POST})
    public void helloRequest(){
        System.out.println("helloRequest");
    }
}
